src/main/java/TaskList.java

```diff
@@ -0,0 +1,62 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskList {
+    private final String SEP = "____________________________________________________________";
```

Comment:
Consider making the constant here "static" if it is not being modified across all instances